pkg/pmem-device-manager/pmd-util.go

```diff
@@ -12,6 +12,9 @@ import (
 )
 
 const (
+	// 1 GB align in ndctl creation request has proven to be reliable.
+	ndctlAlign       uint64        = 1024 * 1024 * 1024
```

Comment:
Is this still the case, As mentioned in related issue https://github.com/pmem/ndctl/issues/76 I guess the proposed patch got merged and that supposed to fix the default 2MB alignment issue. 

Or else what about this proposal: we change our code to create the namespace with default alignment, in case of alignment error, then try with 1GB alignment.

Reply:
It would be relief if improved, we need to re-test this.

Reply:
> create the namespace with default alignment, in case of alignment error, then try with 1GB alignment.

That's likely the way we have to do it, as we can not be sure on top of which kernel code we run, the one before or after fix. If we statically encode use default alignment, we also have to require new kernel, and that is not so welcome requirement.